pkg/cmd/roachtest/tests/backup.go

```diff
@@ -15,6 +15,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
```

Comment:
our linter is probably going to complain that this import should be in the bottom set of imports. Do you have our inhouse formatter `crlfmt` setup, it should be doing this automatically.

Reply:
Yeah, I totally forgot to run it after rebasing with master. Will try to remember to always do that

Reply:
I think you can also configure your IDE to do it on save.